src/main/java/bio/terra/service/auth/iam/IamResourceType.java

```diff
@@ -36,4 +37,32 @@ static IamResourceType fromValue(String text) {
     }
     return null;
   }
+
+  public static IamResourceType toIamResourceType(IamResourceTypeEnum apiEnum) {
+    if (apiEnum == IamResourceTypeEnum.SPEND_PROFILE) {
+      return IamResourceType.SPEND_PROFILE;
+    }
+    return toIamResourceType(apiEnum.toString());
+  }
+
+  public static IamResourceType toIamResourceType(String stringResourceType) {
+    for (IamResourceType b : IamResourceType.values()) {
+      if (String.valueOf(b).equalsIgnoreCase(stringResourceType)) {
+        return b;
+      }
+    }
+    throw new RuntimeException("Invalid resource type: " + stringResourceType);
+  }
+
+  public static IamResourceTypeEnum toIamResourceTypeEnum(IamResourceType resourceType) {
```

Comment:
I would make this a non-static method on `IamResourceType`. You could add `IamResourceTypeEnum` as a constructor argument to the enum type, and then this would return the value of that object instead of using an `if` or `for`. That way you could get rid of the `throw` a well.